controllers/clustergroupupgrade_controller.go

```diff
@@ -873,6 +873,7 @@ func (r *ClusterGroupUpgradeReconciler) copyManagedInformPolicy(
 	labels["app"] = "openshift-cluster-group-upgrades"
 	labels["openshift-cluster-group-upgrades/clusterGroupUpgrade"] = clusterGroupUpgrade.Name
 	labels["openshift-cluster-group-upgrades/parentPolicyName"] = managedPolicy.GetName()
+	labels["velero.io/exclude-from-backup"] = "true"
```

Comment:
make the label name a constant? i know there are many others. i wanted to convert them, just never get the chance while issues kept popping up from scale lab.

Reply:
https://issues.redhat.com/browse/CNF-4919
I added a bullet in this story.

Reply:
@jc-rh do you prefer to define it globally at the top of the file or create another file like constants.go and define it there?

Reply:
Depends on how widely it's used. Let's do all of them together as part of CNF-4919.

Reply:
I added a constant in utils/constant.go